src/amo/components/Link/index.js

```diff
@@ -58,6 +58,8 @@ export class LinkBase extends React.Component {
       prependClientApp,
       prependLang,
       to,
+      target,
+      rel,
```

Comment:
This is probably not needed.

Reply:
If i remove target and rel from line 61, 62. Two of the below tests are failing.
![screen shot 2018-08-22 at 1 25 14 pm](https://user-images.githubusercontent.com/17110163/44450586-f8e04380-a60e-11e8-9058-53c8e693180f.png)

Reply:
Yep, `target` is required, but not `rel`